src/lib/dnssd/minimal_mdns/Server.cpp

```diff
@@ -386,7 +386,9 @@ CHIP_ERROR ServerBase::BroadcastImpl(chip::System::PacketBufferHandle && data, u
             }
             else
             {
-                ChipLogError(Discovery, "Attempt to mDNS broadcast failed:  %s", chip::ErrorStr(err));
+                char ifaceName[20];
+                info->mInterfaceId.GetInterfaceName(ifaceName, sizeof(ifaceName));
+                ChipLogDetail(Discovery, "Warning: Attempt to mDNS broadcast failed on %s:  %s", ifaceName, chip::ErrorStr(err));
```

Comment:
Suggest making a root cause fix where we just don't log the unnecessary errors

Reply:
Root cause fix may be hard: I believe this is because on linux not all interfaces have IP addresses or may not have valid IP addresses for broadcast - not sure which, but I know on my machine this shows for "docker" tuns interfaces, eth without a cable plugged in and local loopback.

If we would have a method to say "detect if interface valid" this can for sure be done. In that case however we should probably still log something like "Broadcast not supported on ....", so log does not go away, we just change the apparent severity of it. Not sure if worth the bother though - if I review logs and it says "cannot broadcast on docker0" I can ignore it.